tests/bootstrap_spec.lua

```diff
@@ -0,0 +1,34 @@
+local a = require "plenary.async_lib.tests"
+
+a.describe("initial start", function()
+  local uv = vim.loop
+  local home_dir = uv.os_homedir()
+  local lvim_config_path = home_dir .. "/.config/lvim"
+  local lvim_runtime_path = home_dir .. "/.local/share/lunarvim/lvim"
+  a.it("should not be reading default neovim directories in the home directoies", function()
+    local rtp_list = vim.opt.rtp:get()
+    local found_illegal_path = vim.tbl_contains(rtp_list, vim.fn.stdpath "config")
+    assert.same(found_illegal_path, false)
+  end)
+  a.it("should be able to read lunarvim directories", function()
+    local rtp_list = vim.opt.rtp:get()
+    assert.is_true(vim.tbl_contains(rtp_list, lvim_runtime_path))
+    assert.is_true(vim.tbl_contains(rtp_list, lvim_config_path))
+  end)
+  a.it("should be able to run treesitter without errors", function()
+    vim.cmd [[TSInstall lua]]
```

Comment:
Would it work to do a pcall(vim.cmd ....) and check the return value ?

Reply:
That should work to check if the command exists or not, but it doesn't look too reliable to me, examples:

```lua
local status, msg = pcall(vim.cmd, "TSInstall lua") -- status -> true
local status, msg = pcall(vim.cmd, "TSInstall gibberish") -- status -> false
local status, msg = pcall(vim.cmd, "checkhealth treesitter") -- status -> true
local status, msg = pcall(vim.cmd, "checkhealth gibberish") -- status -> true ... wtf!
```